universal-login-relayer/src/core/services/EmailService.ts

```diff
@@ -0,0 +1,40 @@
+import nodemailer, {SentMessageInfo} from 'nodemailer';
+import {MailOptions} from 'nodemailer/lib/json-transport';
+import Mail from 'nodemailer/lib/mailer';
+import {CannotSendEmail} from '../utils/errors';
+import {confirmationEmailHtml} from '../utils/confirmationEmailHtml';
+
+export class EmailService {
```

Comment:
I wonder if EmailService should be in the core layer. I would move it to integration.

Reply:
Done